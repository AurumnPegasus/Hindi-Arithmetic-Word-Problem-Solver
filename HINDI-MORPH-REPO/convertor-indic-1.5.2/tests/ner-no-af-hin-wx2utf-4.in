<Sentence id="4">
1	((	NP	<fs etype='timex' esubtype='date' value='19:06:1930'>
1.1	jUna	unk	<fs etype='timex' esubtype='date' value='xx:06:xx'>
1.2	((	NP	<fs etype='numx' value='19.0'>
1.2.1	19	unk	<fs etype='numx' value='19.0'>
	))
1.3	,	unk
1.4	((	NP	<fs etype='numx' value='1930.0'>
1.4.1	1930	unk	<fs etype='numx' value='1930.0'>
	))
	))
2	ke	unk
3	((	NP	<fs etype='timex' esubtype='dayOfWeek'>
3.1	maMgalavAra	unk <fs etype='timex' esubtype='dayOfWeek'>
	))
4	kI	unk
5	((	NP	<fs etype='timex' esubtype='timeOfDay'>
5.1	subaha	unk	<fs etype='timex' esubtype='timeOfDay'>
	))
6	yuxXa	unk
7	ko	unk
8	((	NP	<fs etype='timex' esubtype='timeMeasure' unit='Month value='10.0'>
8.1	((	NP	<fs etype='numx' value='10.0'>
8.1.1	xasa	unk	<fs etype='numx' value='10.0'>
	))
8.2	mahIne	unk	<fs etype='timex' esubtype='timeMeasure' Unit='Month' value='1.0'>
	))
9	pure	unk
10	hoMge	unk
11	.	unk
</Sentence>
