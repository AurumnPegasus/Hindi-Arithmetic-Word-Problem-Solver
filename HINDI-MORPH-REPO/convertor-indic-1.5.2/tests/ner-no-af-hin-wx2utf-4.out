<Sentence id="4">			
1	((	NP	<fs  esubtype='date'  etype='timex'  value='19:06:1930'>
1.1	जून	unk	<fs  esubtype='date'  etype='timex'  value='xx:06:xx'>
1.2	((	NP	<fs  etype='numx'  value='19.0'>
1.2.1	19	unk	<fs  etype='numx'  value='19.0'>
	))
1.3	,	unk	
1.4	((	NP	<fs  etype='numx'  value='1930.0'>
1.4.1	1930	unk	<fs  etype='numx'  value='1930.0'>
	))
	))
2	के	unk	
3	((	NP	<fs  esubtype='dayOfWeek'  etype='timex'>
3.1	मंगलवार	unk <fs etype='timex' esubtype='dayOfWeek'>	
	))
4	की	unk	
5	((	NP	<fs  esubtype='timeOfDay'  etype='timex'>
5.1	सुबह	unk	<fs  esubtype='timeOfDay'  etype='timex'>
	))
6	युद्ध	unk	
7	को	unk	
8	((	NP	<fs  esubtype='timeMeasure'  etype='timex'  value='10.0'  unit='Month'>
8.1	((	NP	<fs  etype='numx'  value='10.0'>
8.1.1	दस	unk	<fs  etype='numx'  value='10.0'>
	))
8.2	महीने	unk	<fs  esubtype='timeMeasure'  etype='timex'  value='1.0'  Unit='Month'>
	))
9	पुरे	unk	
10	होंगे	unk	
11	.	unk	
</Sentence>			
